source/exec.tex

```diff
@@ -5672,3 +5690,845 @@
+\pnum
+The type \exposid{own-env-t} is \tcode{Environment::template
+env_type<decltype(get_env(\linebreak declval<Rcvr>()))>} if that % avoid Overfull
+qualified-id is valid and denotes a type, \tcode{env<>} otherwise.
+\end{itemdescr}
+
+\begin{itemdecl}
+template<class R>
+@\exposid{state}@(coroutine_handle<promise_type> h, R&& rr);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes
+
+\item \exposid{handle} with \tcode{std::move(h)};
+\item \exposid{rcvr} with \tcode{std::forward<R>(rr)};
+\item \exposid{own-env}
+    with \tcode{\exposid{own-env-t}(get_env(\exposid{rcvr}))} if that expression
+    is valid and \tcode{\exposid{own-env-t}()} otherwise.
+    If neither of these expressions is valid, the program is ill-formed.
+\item \exposid{environment} with \tcode{Environment(\exposid{own-env})} if that expression is
+    valid, otherwise \tcode{Environment(get_env(\exposid{rcvr}))}
+    if this expression is valid, otherwise \tcode{Environment()}.
+    If neither of these expressions is valid, the program is ill-formed.
+\end{itemdescr}
+
+\begin{itemdecl}
+~@\exposid{state}@();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+
+\begin{codeblock}
+if (@\exposid{handle}@)
+  @\exposid{handle}@.destroy();
+\end{codeblock}
+\end{itemdescr}
+
+\begin{itemdecl}
+void start() & noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Let \tcode{prom} be the object \tcode{\exposid{handle}.promise()}.
+Associates \tcode{STATE(prom)}, \tcode{RCVR(prom)}, and \tcode{SCHED(prom)}
+with \tcode{*this} as follows:
+
+\begin{itemize}
+\item \tcode{STATE(prom)} is \tcode{*this}.
+\item \tcode{RCVR(prom)} is \exposid{rcvr}.
+\item \tcode{SCHED(prom)} is the object initialized
+with \tcode{scheduler_type(get_scheduler(get_env(\exposid{rcvr})))}
+if that expression is valid and \tcode{scheduler_type()} otherwise.
+If neither of these expressions is valid, the program is ill-formed.
+\end{itemize}
+
+Let \tcode{st} be \tcode{get_stop_token(get_env(\exposid{rcvr}))}.
+Initializes \tcode{prom.\exposid{token}} and
+\tcode{prom.\exposid{source}} such that
+
+\begin{itemize}
+\item
+\tcode{prom.\exposid{token}.stop_requested()} returns
+\tcode{st.stop_requested()};
+\item
+\tcode{prom.\exposid{token}.stop_possible()} returns
+\tcode{st.stop_possible()}; and
+\item
+for types \tcode{Fn} and \tcode{Init} such that both
+\tcode{\libconcept{invocable}<Fn>} and
+\tcode{\libconcept{constructible_from}<Fn, Init>} are modeled,
+\tcode{stop_token_type::callback_type<Fn>} models
+\tcode{\exposid{stoppable-callback-for}<Fn,\linebreak stop_token_type, % avoid Overfull
+Init>}.
+\end{itemize}
+
+After that invokes \tcode{\exposid{handle}.resume()}.
+\end{itemdescr}
+
+\rSec3[task.promise]{Class \tcode{task::promise_type}}
+
+\begin{codeblock}
+namespace std::execution {
+  template<class E>
+  struct with_error {
+    using type = remove_cvref_t<E>;
+    type error;
+  };
+  template<class E>
+  with_error(E) -> with_error<E>;
```

Comment:
```suggestion
  template<class E>
    with_error(E) -> with_error<E>;
```
In general, there should be two spaces of indentation after a *template-head*, except for class definitions. I won't make any more comments for this.

Reply:
I applied the change with [this commit](https://github.com/cplusplus/draft/pull/8032/commits/70456ebe2b2681393bf3da89887c1c1e48356150).